### ISSUE TITLE
Loading tests with Gatling

### ISSUE THREAD

**Issue by UO270115** [Arquisoft/radarin_en2b#115]

**Comment by pglez82** [comment]
Sorry. I forgot that I have a plugin installed in firefox that allows CORS!:

https://mybrowseraddon.com/access-control-allow-origin.html

With that, it should work.

**Comment by UO270115** [comment]
I have downloaded and enable CORS as:

![image](https://user-images.githubusercontent.com/60620700/115306383-09eb0500-a168-11eb-95a2-4bb0027310d5.png)

but with the proxy configured as previously shown the test CORS is: 

![image](https://user-images.githubusercontent.com/60620700/115306430-1b341180-a168-11eb-96d9-06a89a5efe8e.png)

and the requests are still blocked, I do not know why @pglez82 

![image](https://user-images.githubusercontent.com/60620700/115306531-474f9280-a168-11eb-8912-2123de7e6ea7.png)

**Comment by pglez82** [comment]
Ok. I tried uninstalling my cors plugin and everything works fine anyway. One question, does radarin0 work for you without using the gatling proxy?

**Comment by UO270115** [comment]
Yes, without the firefox proxy configured (default) and without allow CORS extension on, both applications (Radarin 0 and RadarinEn2b) works perfectly as normal:

![image](https://user-images.githubusercontent.com/60620700/115386149-54589a00-a1d9-11eb-81aa-4e0e6d7be0f0.png)

![image](https://user-images.githubusercontent.com/60620700/115386951-52dba180-a1da-11eb-814f-7802389999f2.png)

The only thing is that the Gatling recorder is not recording anything.

**Comment by pglez82** [comment]
I think I have missed one step in my explanation on monday:

https://github.com/Arquisoft/radarin_0/tree/master/webapp#load-testing-gatling

Step 4. After generating the certificates, we must import the certificate to firefox.

Obviously as I prepared this at the beginning of the course, I already had it in my firefox... Let me know if this solves it.

**Comment by UO270115** [comment]
It is still the same. I am sure that is my mistake and a silly one. I will ask some of my mates to try it just in case and for now I will start doing something else. Thanks.

**Comment by UO270185** [comment]
We managed to record everything with Gatling by adding inrupt.net to the pages not used by the proxy, as it seemed that the problem was the proxy blocking some requests.
